backend/dal/Services/AccessRequest.cs

```diff
@@ -189,10 +197,12 @@ public AccessRequest Update(AccessRequest update)
             // Copy values into entity.
             accessRequest.RoleId = update.RoleId;
             accessRequest.RowVersion = update.RowVersion;
```

Comment:
Make sure concurrency works.  I know just setting the value like this normally does not.